### ISSUE TITLE
When listview dynamically loops, subcomponent @tr will not be translated

### ISSUE THREAD

**Issue by CarbonPool** [slint-ui/slint#4378]
`scroll := ListView {
        padding: 0;
        for wifi in root.wifi-list : WifiItem {
               ssid: wifi.ssid;
                       moved(mouse-y, pressed-y) => {
                             scroll.viewport-y += mouse-y - pressed-y;
                        }
                        clicked => {
                                   root.wifi-passwd = "";
                                   root.show-connect-wifi-dlg = true;
                                   root.select-wifi-ssid = wifi.ssid;
                         }
        }
}`

**Comment by CarbonPool** [comment]
#WifiItem
`import { Button } from "std-widgets.slint";
export component WifiItem {
    callback clicked();
    callback moved(length, length);
    in property <string> ssid;
    HorizontalLayout {
        padding-bottom: 10px;
        Rectangle {
            TouchArea {
                moved => {
                    root.moved(self.mouse-y, self.pressed-y);
                }
            }
            HorizontalLayout {
                Text {
                    text: root.ssid;
                    horizontal-alignment: left;
                    vertical-alignment: center;
                    font-size: 16px;
                }
            }
        }
        HorizontalLayout {
            Button {
                width: 80px;
                text: @tr("connect");
                clicked => {
                    root.clicked();
                }
            }
        }
    }
}

export struct WifiItemModel {
    ssid: string
}`

@tr("connect") not working

**Comment by ogoffart** [comment]
could you elaborate what do you mean by not working?
In which context? what platform/version of slint/...
Do other tr works?

**Comment by CarbonPool** [comment]
> could you elaborate what do you mean by not working? In which context? what platform/version of slint/... Do other tr works?

WifiItem.slint @tr("connect") cannot be translated normally, fixed mapping text, if app.slint contains @tr("connect"), and then update the translation po file, the translation will be displayed normally

**Comment by ogoffart** [comment]
do you run the tr-extractor on all the files?

**Comment by CarbonPool** [comment]
> do you run the tr-extractor on all the files?

Yes, I refer to: "find -name \*.slint | xargs slint-tr-extractor -o MY_PROJECT.pot"

**Comment by ogoffart** [comment]
And it did not extract the text for the "connect" string?
If i paste your code in a .slint file, it extracts it properly for me

**Comment by CarbonPool** [comment]
I updated the file from poetry, I created a small example but it doesn't seem to reproduce the problem again, I'm not sure if it's a bug in the conversion tool or the translation tool.
